Header header
uint32 xsize=4
uint32 ysize=4
float32[] calibration
